packages/immutadot-benchmark/src/benchmark.js

```diff
@@ -0,0 +1,55 @@
+export function createBenchmark(title, testResult, pMaxTime = 30, pMaxOperations = 1000) {
+
+  const fast = Boolean(process.env.FAST)
+  const maxTime = fast ? pMaxTime / 3 : pMaxTime, maxOperations = fast ? Math.round(pMaxOperations / 3) : pMaxOperations
+
+  const runs = []
+
+  function run(key, opTitle, operation) {
```

Comment:
Maybe you could add more comments to explain more your approach ? Or a pointer on the method you followed ?

Reply:
I'll document in a later PR, created an issue #223